pyproject.toml

```diff
@@ -4,7 +4,92 @@ requires = [
   "setuptools>=50",
   "setuptools_scm>=6",
   "wheel>=0.34",
-]  # ought to mirror 'requirements/build.txt'
+]
+
+[project]
+name = "xrtpy"
+readme = "README.md"
+keywords = ["solar physics"]
+description = "For analyzing data from the X-Ray Telescope (XRT) on the Hinode spacecraft."
+license = {file = "LICENSE"}
+classifiers = [
+  "Development Status :: 4 - Beta",
```

Comment:
I changed this from `3 - Alpha` to `3 - Beta` since XRTpy has been gaining maturity.